### PR TITLE
Enable windows

### LINKED ISSUE #3: Add Windows
Currently Windows builds are disable on CI. This was due to to missing packages lower in the stack like `cuda-cudart. Now that those are becoming available, it should be possible to enable Windows. Would like to produce packages here for Windows.

### PR DESCRIPTION
**Pull request by adibbley**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

Fixes #3

### CODE REVIEW

**Comment by adibbley** [comment]
@conda-forge-admin, please rerender

**Comment by adibbley** [comment]
Requires https://github.com/conda-forge/cuda-libraries-dev-feedstock/pull/5

**Comment by adibbley** [comment]
@conda-forge-admin , please restart ci

**Comment by adibbley** [comment]
@conda-forge-admin , please restart ci

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,11 +10,11 @@ source:
 
 build:
   number: 0
-  skip: true  # [osx or win]
+  skip: true  # [osx]
 
 requirements:
   run:
-    - cuda-nsight 12.0.78               # [not aarch64]
+    - cuda-nsight 12.0.78               # [linux64 or ppc64le]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Is there a reason to change this?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,11 +10,11 @@ source:
...
     - nsight-compute 2022.4.0.15
     - cuda-nvml-dev 12.0.76
     - cuda-nvvp 12.0.90                 # [not aarch64]
```
Comment (jakirkham) [inline recipe/meta.yaml:20]: Also noticing this now doesn't match. Should we keep them the same?

**Review by jakirkham — COMMENTED** [review]
Thanks Alex! 🙏

Had a question below

File: recipe/meta.yaml (hunk 1 above)
Comment (adibbley) [reply recipe/meta.yaml]: Yes cuda-nsight is not supported on Windows so it needs some change
https://github.com/conda-forge/cuda-nsight-feedstock/blob/main/recipe/meta.yaml#L14-L15

File: recipe/meta.yaml (hunk 2 above)
Comment (adibbley) [reply recipe/meta.yaml:20]: `cuda-nvvp` is supported on windows, so there is a difference here
https://github.com/conda-forge/cuda-nvvp-feedstock/blob/main/recipe/meta.yaml#L16-L18

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml]: How about?

```suggestion
    - cuda-nsight 12.0.78               # [not (aarch64 or win)]
```

**Review by jakirkham — APPROVED** [review]
Thanks Alex! 🙏

**Pull request merged**